ddtrace/internal/ci_visibility/recorder.py

```diff
@@ -72,6 +93,36 @@ def __init__(self, tracer=None, config=None, service=None):
         except Exception:
             log.warning("Failed to load CODEOWNERS", exc_info=True)
 
+    def _check_enabled_features(self):
+        # type: () -> Tuple[bool, bool]
+        if not self._app_key:
+            return False, False
+        url = "https://api.datadoghq.com/api/v2/libraries/tests/services/setting"
+        _headers = {"dd-api-key": os.getenv("DD_API_KEY"), "dd-application-key": self._app_key}
+        payload = {
+            "data": {
+                "id": str(uuid4()),
+                "type": "ci_app_test_service_libraries_settings",
+                "attributes": {
+                    "service": self._service,
+                    "env": ddconfig.env,
+                    "repository_url": self._tags.get(ci.git.REPOSITORY_URL),
+                    "sha": self._tags.get(ci.git.COMMIT_SHA),
+                    "branch": self._tags.get(ci.git.BRANCH),
+                },
```

Comment:
there are some extra configuration parameters here that we might want to add (it's ok to add at a later PR though):
```python
                    "configurations": {
                         "runtime.name": "CPython",
                         "runtime.version": "<version>",
                         # ... some others
                    }
                },
```

Reply:
@juan-fernandez What is the purpose of attaching these extra configuration parameters into the payload? I thought this is just to check the repo/sha/branch combination?

Reply:
that's a fair question. I know other tracers include it but I forwarded the question to our backend team, because I don't know the reason